integration-test/concat/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mediachain/concat-dependencies
+
+# Clone concat repo
+RUN mkdir -p /go/src/github.com/mediachain
+RUN rm -rf /go/src/github.com/mediachain/concat
+RUN git clone https://github.com/mediachain/concat /go/src/github.com/mediachain/concat
+WORKDIR /go/src/github.com/mediachain/concat
+ARG concat_ref=master
+RUN git checkout ${concat_ref}
+
+# install gx dependencies
+RUN gx install --global
```

Comment:
Is this still nevessary here?
We already fetch the deps in the base image, and updating these deps are discrete events (only when we update libp2p).